app/config/tstvhq/rooms.json

```diff
@@ -261,7 +261,10 @@
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
             },
             "directions": {
-                "south": "theater"
+                "south": "
+                
+                
```

Comment:
this is definitely wrong, was this an accident?

Reply:
I believe I've just fixed it but let me double check again.